src/openrct2/drawing/Font.cpp

```diff
@@ -502,93 +484,98 @@ bool font_supports_string_sprite(const utf8* text)
         bool supported = false;
         switch (codepoint)
         {
-            case FORMAT_ENDQUOTES:
-            case FORMAT_UP:
-            case FORMAT_INVERTEDEXCLAMATION:
-            case FORMAT_POUND:
-            case FORMAT_YEN:
-            case FORMAT_COPYRIGHT:
-            case FORMAT_DOWN:
-            case FORMAT_LEFTGUILLEMET:
-            case FORMAT_TICK:
-            case FORMAT_CROSS:
-            case FORMAT_RIGHT:
-            case FORMAT_DEGREE:
-            case FORMAT_SYMBOL_RAILWAY:
-            case FORMAT_SQUARED:
-            case FORMAT_OPENQUOTES:
-            case FORMAT_EURO:
-            case FORMAT_SYMBOL_ROAD:
-            case FORMAT_SYMBOL_FLAG:
-            case FORMAT_APPROX:
-            case FORMAT_POWERNEGATIVEONE:
-            case FORMAT_BULLET:
-            case FORMAT_RIGHTGUILLEMET:
-            case FORMAT_SMALLUP:
-            case FORMAT_SMALLDOWN:
-            case FORMAT_LEFT:
-            case FORMAT_INVERTEDQUESTION:
-
-            case UNICODE_A_OGONEK_UC:
-            case UNICODE_C_ACUTE_UC:
-            case UNICODE_E_OGONEK_UC:
-            case UNICODE_N_ACUTE_UC:
-            case UNICODE_L_STROKE_UC:
-            case UNICODE_S_ACUTE_UC:
-            case UNICODE_Z_DOT_UC:
-            case UNICODE_Z_ACUTE_UC:
-
-            case UNICODE_A_OGONEK:
-            case UNICODE_C_ACUTE:
-            case UNICODE_E_OGONEK:
-            case UNICODE_N_ACUTE:
-            case UNICODE_L_STROKE:
-            case UNICODE_S_ACUTE:
-            case UNICODE_Z_DOT:
-            case UNICODE_Z_ACUTE:
-
-            case UNICODE_CYRILLIC_IO_UC:
-            case UNICODE_CYRILLIC_IO:
-
-            case FORMAT_GERMAN_OPENQUOTES:
-            case FORMAT_SINGLE_OPENQUOTE:
-            case FORMAT_SINGLE_ENDQUOTE:
-            case FORMAT_GERMAN_SINGLE_OPENQUOTE:
-
-            case UNICODE_GUILDER_SIGN:
-
-            case UNICODE_G_BREVE_UC:
-            case UNICODE_I_WITH_DOT_UC:
-            case UNICODE_S_CEDILLA_UC:
-            case UNICODE_G_BREVE:
-            case UNICODE_I_WITHOUT_DOT:
-            case UNICODE_S_CEDILLA:
-
-            case UNICODE_INTERPUNCT:
-            case UNICODE_ELLIPSIS:
-
-            case UNICODE_A_BREVE_UC:
-            case UNICODE_A_BREVE:
-            case UNICODE_S_COMMA_UC:
-            case UNICODE_S_COMMA:
-            case UNICODE_T_COMMA_UC:
-            case UNICODE_T_COMMA:
-
-            case UNICODE_C_CARON_UC:
-            case UNICODE_C_CARON:
-            case UNICODE_Y_ACUTE_UC:
-            case UNICODE_Y_ACUTE:
-
-            case UNICODE_O_DOUBLE_ACUTE_UC:
-            case UNICODE_O_DOUBLE_ACUTE:
-            case UNICODE_U_DOUBLE_ACUTE_UC:
-            case UNICODE_U_DOUBLE_ACUTE:
+            // Latin alphabet
```

Comment:
I was hoping we could ditch this switch block as well. It could use the same map and binary search algorithm I proposed earlier.